todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskContract.java

```diff
@@ -39,9 +39,7 @@
 
     interface Presenter extends BasePresenter {
 
-        void createTask(String title, String description);
-
-        void updateTask( String title, String description);
+        void saveTask(String title, String description);
```

Comment:
I like that idea of these contracts being simpler, even if that gives more work to @JoseAlcerreca for merging 👍